src/lib/__tests__/total.test.js

```diff
@@ -4,14 +4,11 @@ describe("total", () => {
   const loader = jest
     .fn()
     .mockReturnValue(Promise.resolve({ headers: { "x-total-count": "50" } }))
-  it("loads the path and passes in the token", () => {
-    return totalViaLoader(loader, {}, { size: 10, extra_option: 1 }).then(
-      total => {
-        expect(loader.mock.calls[0][0].size).toBe(0)
-        expect(loader.mock.calls[0][0].total_count).toBe(true)
-        expect(loader.mock.calls[0][0].extra_option).toBe(1)
-        expect(total).toBe(50)
-      }
-    )
-  })
+  it("loads the path and passes in the token", () =>
```

Comment:
Is prettier stripping out the `{` from `it("loads the path and passes in the token", () =>`? If not, implicit returns are hard to maintain across multiple lines and should be avoided.

Reply:
I think we added eslint-prettier after this, that said - this test could probably just be an async test, would be easier to read too

Reply:
So, I tweaked the formatting without changing the structure, if this helps? I find the multiline implicit returns really readable if they're indented in a way that reflects that flow.

If we don't want to do multiline implicit returns then we'll need a custom lint rule for that, probably, rather than trying to catch them one-at-a-time in a big PR like this?

Reply:
@mbilokonsky - I think its a good rule to of thumb to try to avoid non-automated structural changes during refactors. Would you mind re-adding brackets? Missing brackets tend to introduce fragility over the long-term. (We can develop custom rules when necessary but that's for a different PR.)

Reply:
but this was an automated change, driven by the listing rules and (probably?) implemented via `lint --fix`. Does that change your request? 

Even just scanning the files changed in this PR shows me half a dozen similar changes in the first few files. If we as an institution care about implicit vs explicit returns I'm happy to add a lint rule around that, but if it's personal preference then I'd rather have that conversation in a subsequent PR? So, sort of the opposite of what you're saying, I guess?

Reply:
Ah, I thought this was a manual change 👍. Would you mind turning off that rule so that the structure remains the same, with brackets, and rerun `lint --fix`?

Reply:
So, I tried that and it left me with a new 500+ linting errors and I'm not willing to make that change right now.

This is the "Our ESLint toolchain was brought into the present with some rough attempts at sensible defaults" PR. 

I don't want to suggest that your preference is invalid here, but I do suggest that stuff like this be moved to subsequent "I want a different rule" PRs that we can evaluate and apply on a case-by-case basis.

Is that reasonable?

Reply:
There were brackets before, and now there are not brackets. Since code is more stable with them over time, and there's existing precident, and additionally since this change was automated (meaning it can be undone easily), I don't think this is that controversial, or even in need of discussion. 

I can pull down your branch and take a look if you'd like?